app/classes/Transvision/Project.php

```diff
@@ -231,7 +255,12 @@ public static function getLocaleInContext($locale, $context)
         }
 
         // Firefox for iOS: no mapping
-        $locale_mappings['firefox_ios'] = [];
+        $locale_mappings['firefox_ios'] = [
```

Comment:
I'm just doing a preliminary swap of the code, but I don't understand this change.

We do have 3 [different translations for Spanish](https://l10n.mozilla-community.org/~flod/webstatus/?product=firefox-ios): es, es-CL, es-MX. With this we'll lose es-CL and es-MX.